solidity/integration/stress_test.js

```diff
@@ -0,0 +1,96 @@
+const BondedECDSAKeepVendor = artifacts.require("./BondedECDSAKeepVendor.sol")
+const BondedECDSAKeepVendorImplV1 = artifacts.require(
+  "./BondedECDSAKeepVendorImplV1.sol"
+)
+const BondedECDSAKeepFactory = artifacts.require("./BondedECDSAKeepFactory.sol")
+const BondedECDSAKeep = artifacts.require("./BondedECDSAKeep.sol")
+
+// how many iterations
+const iterations = 10
+// delay between the iterations [ms]
+const delay = 30000
+// number of keeps created at once
+const stressLevel = 5
+
+const groupSize = 3
+const threshold = 3
+const bond = 10
+
+module.exports = async function () {
+  const accounts = await web3.eth.getAccounts()
+
+  const keepOwner = accounts[4]
+  const application = accounts[5]
```

Comment:
We added `accounts[4]` to operators list, so maybe want to replace `keepOwner` here?
```suggestion
  const keepOwner = accounts[5]
  const application = accounts[6]
```

Reply:
Right! Done in https://github.com/keep-network/keep-ecdsa/pull/357/commits/650a589fc64e9d984bfbac0eeeee1ca1f37fe1fb